src/hosted-buttons/utils.test.js

```diff
@@ -309,4 +315,99 @@ describe("buildHostedButtonOnApprove", () => {
   });
 });
 
+test("getFlexDirection", () => {
+  expect(getFlexDirection({ layout: "horizontal" })).toStrictEqual({
+    flexDirection: "row",
+  });
+  expect(getFlexDirection({ layout: "vertical" })).toStrictEqual({
+    flexDirection: "column",
+  });
+});
+
+test("getButtonColor", () => {
+  const colors = ["gold", "blue", "silver", "white", "black"];
+  const fundingSources = ["paypal", "venmo", "paylater"];
+  const colorMap = {
```

Comment:
option to export this object from `utils.js` and then reuse it here

Reply:
I thought about reusing that color map, but I was concerned about someone accidentally changing a value. If the tests were using the same object, then they would automatically pass.

Happy to still do make the change, but that was my thought process.

Reply:
clever!